lib/install/management/appliance.go

```diff
@@ -323,8 +323,8 @@ func (d *Dispatcher) createApplianceSpec(conf *config.VirtualContainerHostConfig
 			GuestId:            string(types.VirtualMachineGuestOsIdentifierOtherGuest64),
 			AlternateGuestName: constants.DefaultAltVCHGuestName(),
 			Files:              &types.VirtualMachineFileInfo{VmPathName: fmt.Sprintf("[%s]", conf.ImageStores[0].Host)},
-			NumCPUs:            int32(vConf.ApplianceSize.CPU.Limit),
-			MemoryMB:           vConf.ApplianceSize.Memory.Limit,
+			NumCPUs:            int32(*vConf.ApplianceSize.CPU.Limit),
```

Comment:
technically we should have nil checks if these are now nilable.

Reply:
_technically_ I guess I agree...but we _know_ it's always set based on the appliance validation logic...